tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+min_version = 4
+# list of environments to run by default with tox run
+env_list =
+    lint,
+    compile,
+    py312-docs,
+    py3132-tests,
+    a11y-tests,
+
+# general tox env configuration
+# these can be run with any py3{9,12} version
+# tox run -e py39-lint  
+[testenv]
+deps =
+    lint: pre-commit
+    compile: sphinx-theme-builder[cli]
+    profile-docs: py-spy
```

Comment:
I left here tasks that are either needed by others (like compile) and general maintenance tasks (lint, profile-docs)